packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,471 @@
+            background: var(--aa-background-color);
+        }
+        // Line separator
+        .aa-SourceHeaderLine {
+            z-index: 1;
+            position: absolute;
+            display: block;
+            top: var(--aa-spacing-half);
+            left: 0;
+            right: 0;
+            height: 2px;
+            opacity: 0.3;
+            border-bottom: solid 1px var(--aa-primary-color);
+        }
+        // hide empty header
+        &:empty {
+            display: none;
+        }
+    }
+    &:empty {
+        // hide empty section
+        display: none;
+    }
+    // the list of results inside the source
+    ul.aa-List {
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
 }
 
-.aa-Input,
-.aa-TouchSearchButton {
-  border: 1px solid #d6d6e7;
-  border-radius: 3px;
-  box-shadow: rgb(119 122 175 / 30%) 0 1px 4px 0 inset;
-  font: inherit;
+// ----------------
+// Hit Layout - smart layout
+// ----------------
+li.aa-Item {
+    display: flex;
+    padding: 0 var(--aa-spacing-half);
+    line-height: 1.1em;
+    align-items: center;
+    color: var(--aa-text-color);
+    height: 100%;
+    cursor: pointer;
+    // when the result is active
+    &[aria-selected='true'] {
+        background-color: var(--aa-selected-color);
+        .aa-ItemActionButton {
+            visibility: visible;
+        }
+    }
+    // wrap hit with url but we don't need to see it
+    a.aa-ItemLink {
+        color: inherit;
+        text-decoration: none;
+        width: 100%;
+        display: flex;
+        align-items: center;
+    }
+    // the result type icon inlined svg or img
+    .aa-ItemSourceIcon {
+        box-shadow: inset 0 0 0 1px var(--aa-selected-color);
+        border-radius: 3px;
+        margin: 2px 0 2px 2px;
+        flex-shrink: 0;
+        font-size: var(--aa-icon-size);
+        width: calc(var(--aa-icon-size) + var(--aa-spacing));
+        height: calc(var(--aa-icon-size) + var(--aa-spacing));
+        stroke-width: var(--aa-icon-stroke-width);
+        color: var(--aa-icon-color);
+        margin-right: var(--aa-spacing-half);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
+        img,
+        svg {
+            width: var(--aa-icon-size);
+            height: var(--aa-icon-size);
+        }
+    }
+    .aa-ItemContent {
+        flex-shrink: 1;
+        overflow: hidden;
+        width: 100%;
+        color: var(--aa-text-color);
+        cursor: pointer;
+        padding: calc(var(--aa-spacing) / 4) 0;
+        .aa-ItemContentTitle {
+            display: inline-block;
+            max-width: 100%;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+            overflow-x: hidden;
+            mark {
+                color: var(--aa-text-color);
+                background: none;
+                font-style: normal;
+                font-weight: bold;
+            }
```

Comment:
We could move this to `.aa-Item` for everybody to get this style even when not respecting the HTML tree structure. Wdyt?

Reply:
Might need a different highlightinh style on other elements than title, but some properties can be shared yes.